sdk/communication/test-resources.json

```diff
@@ -46,9 +72,25 @@
     }
   ],
   "outputs": {
+    "AZURE_SUBSCRIPTION_ID": {
```

Comment:
i don't think this is needed.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/15105/files#r624324055
Conflicting comments. But I think we are discussing it offline and agree subscription id is not needed